### PR TITLE
Feat project page

### PR DESCRIPTION
**Pull request by austinmargarone**

### CODE REVIEW

**Comment by brandonetter** [comment]
Do you want me to merge this? We discussed fixing the design image to extend to the full width of the screen. You can either set the width of all it's containers to full (currently it's getting shortened because of it's parent sections), and then you can stretch it to full:

Or you can make it absolute to "break it out" of the width restrictions- just be mindful of positioning.

File: components/projects/Design.tsx [hunk 1]
```diff
@@ -6,14 +7,19 @@ const Design = (props: Props) => {
...
-        <div className="h-[13.08294rem] w-full">
-          {/* <Image src="/Figma.svg" fill={true} alt="Computer" /> */}
-        </div>
       </div>
-      <div></div>
+      <div>
+        <Image
+          src="/Figma.svg"
+          width={393}
+          height={209}
```
Comment (brandonetter) [inline components/projects/Design.tsx]: https://drive.google.com/file/d/1qSEEfn02wqwopeFK9HZksAzeWi8iUcKg/view?usp=sharing

**Review by brandonetter — CHANGES_REQUESTED** [review]

**Comment by austinmargarone** [comment]
@brandonetter I had just ran out of time, working on it now. Thanks for sending the video.

**Comment by austinmargarone** [comment]
@brandonetter How does this look? I set different heights at varying breakpoints. I'd like to throw the breakpoints into globals but am not sure how to transfer the tailwind syntax to CSS.
ex. turn md:max-w-[70rem] into CSS

**Comment by austinmargarone** [comment]
Can you also take a look at this for me? The heading in OtherCases.tsx is not staying within the breakpoints and I'm not sure why. 
Screenshot from 2023-09-21 10-36-28

**Comment by brandonetter** [comment]
@austinmargarone 

https://github.com/austinmargarone/portfolio/assets/4108484/eb7aced2-751c-4201-bf14-2e6c6e9b50bd

Not sure what you're asking about. Your breakpoints are working. I added a blue and red border to the header for the different classes. What do you expect to happen and I can help you.

**Comment by austinmargarone** [comment]
@brandonetter 
The one heading Other Case Studies I believe should be inline with the cards. On larger devices it drifts off to the left. 
Screenshot from 2023-09-22 09-35-23
Screenshot from 2023-09-22 09-35-00
Screenshot from 2023-09-22 09-34-48

**Comment by austinmargarone** [comment]
@brandonetter 
Never mind I got it to look a lot better with padding.

**Comment by brandonetter** [comment]
Great @austinmargarone

**Pull request merged**